app/build.gradle

```diff
@@ -157,7 +157,7 @@ afterEvaluate {
     project.tasks.matching { it.name.startsWith('jaxRs2Retrofit') }.each {
         it.inputDir = file(project(':../fablab-common').projectDir.toString() + '/src/main/java')
         it.outputDir = generatedSourcesDir
-        it.packageName = 'de.fau.cs.mad.fablab.rest.myapi'
+        it.packageName = 'de.fau.fablab.app.rest.myapi'
```

Comment:
OK, here is a source: http://stackoverflow.com/a/17582393
We would thereby create a NEW app, that cannot be installed "over" the old one.